pkgs/games/freeorion/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchgit, cmake, boost, SDL2, python2, freetype, openal, libogg, libvorbis, zlib, libpng, libtiff, libjpeg, mesa, glew, git, doxygen }:
+
+stdenv.mkDerivation rec {
+  version = "2016-04-16";
+  name = "freeorion-${version}";
+
+  src = fetchgit {
+    url = "https://github.com/freeorion/freeorion";
```

Comment:
I guess adding just the patch would keep the game multiplayer compatible with 0.4.5? I guess that would be nice.